src/app/services/project_service.js

```diff
@@ -292,6 +330,32 @@ angular
         });
         return res;
     }
+    
+    function prepareModelsForSearching(models) {
```

Comment:
I didn't know that fuse needs to operate on lists instead of objects... fair enough!

Reply:
pending the answer to the question below, I do wonder if we could just do something like this though:
```
var modelList = _.map(models, function(model) {
  // make a copy
  var newModel = _.assign({}, model);
  newModel.columns = _.values(model.columns);
  return newModel;
});

Reply:
> I didn't know that fuse needs to operate on lists instead of objects

As far as I can tell, you have to specifically name the fields you want to search in the `keys` argument, which is a big pain if each column has its own name instead of just being a generic array member. Could be wrong though! 

> just do something like this
This is exactly the sort of thing I hoped existed. I'm going to just use it and ignore the null vs blank thing until doing so causes trouble.